Sources/XMTP/ApiClient.swift

```diff
@@ -9,8 +9,18 @@ import GRPC
 import XMTPProto
 
 typealias PublishResponse = Xmtp_MessageApi_V1_PublishResponse
+typealias QueryResponse = Xmtp_MessageApi_V1_QueryResponse
+
+protocol ApiClient {
+	var environment: Environment { get }
+	init(environment: Environment, secure: Bool) throws
+	func setAuthToken(_ token: String)
```

Comment:
Today I learned about omitting argument labels in Swift tyty! Makes total sense here too.